shell/platform/embedder/embedder.h

```diff
@@ -41,205 +41,208 @@ typedef enum {
   kSoftware,
 } FlutterRendererType;
 
-// Additional accessibility features that may be enabled by the platform.
```

Comment:
Was this line supposed to be removed? Seems like it was useful.

Reply:
No. Fixed.